apps/prairielearn/src/ee/pages/institutionAdminCourse/institutionAdminCourse.ts

```diff
@@ -39,4 +43,52 @@ router.get(
   }),
 );
 
+const UpdateEnrollmentLimitsBodySchema = z.object({
+  __action: z.literal('update_enrollment_limits'),
+  yearly_enrollment_limit: z.union([z.literal(''), z.coerce.number().int()]),
```

Comment:
We could immediately interpret `''` as `null` to avoid having to remember later?
```suggestion
  yearly_enrollment_limit: z.union([z.literal('').transform(() => null), z.coerce.number().int()]),
```
In PT I've been trying to define some helper schemas for parsing `body`. The above looks like it would be a great `IntegerOrNullInputSchema` (or some better name).

See, for example, https://github.com/PrairieLearnInc/PrairieTest/blob/50c71a751d103b5bd272910d8d14e1fd57c18108/src/lib/zod-types.ts#L18 and other schemas in that file. These need some docstrings but they are all used for parsing `body`.

Reply:
I like this! I'll avoid lifting this into a shared schema for now, but it'd be nice to develop a collection of utilities like that (`@prairielearn/zod`?).

Reply:
Done in bbe856a.

Reply:
A `@prairielearn/zod` lib sounds excellent!